Description: Fix RSpec configuration
 RSpec 3.0 uses config.color instead of config.color_enabled
Last-Update: 2015-05-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -53,7 +53,7 @@
 
 RSpec.configure do |config|
   config.order = :rand
-  config.color_enabled = true
+  config.color = true
 
   config.expect_with :rspec do |expectations|
     expectations.syntax = :expect
